src/main/java/study/car/race/Car.java

```diff
@@ -0,0 +1,18 @@
+package study.car.race;
+
+public class Car {
+
+    private static final int CRITERIA = 4;
+
+    private int distance;
+
+    public void moveOrStop(int number) {
+        if (number >= CRITERIA) {
+            this.distance++;
```

Comment:
중요한 행위는 메서드를 통한 네이밍으로 구분지어두면 가독성이 더 올라갈 수 있겠죠? 

```suggestion
            move();
```